VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -293,7 +293,7 @@ final internal class ResponseCache
             
             let fileURL = directoryURL.URLByAppendingPathComponent(key)
```

Comment:
Can we use a guard let here for fileURL and use fatalError() if fileURL is nil (similar to what cachesDirectoryURL() does above).  That way we can get a more descriptive error instead of a forced unwrap crash if fileURL does end up being nil.